src/nvim/api/vim.c

```diff
@@ -668,3 +669,47 @@ static void write_msg(String message, bool to_err)
   --no_wait_return;
   msg_end();
 }
+
+/// Setup a new namepsace for holding your marks
+///
+/// @param namespace String name for the namespace
+/// @param[out] err Details of an error that may have occurred
+/// @return integer id to be used with future mark_ calls, or 0 if name exists
+Integer nvim_init_mark_ns(String namespace, Error *err)
```

Comment:
nvim_mark_init_ns

"nvim" is not a namespace, it's the standard prefix of all API functions (this will be the case after @bfredl's changes)